docs/scss/components/_example.scss

```diff
@@ -66,6 +66,10 @@ rh-alert + .example {
     }
   }
 
+  &--component {
```

Comment:
Can you do this with using sass concat or bem? We're trying to move away from those